features/270-next-skycli/commands.md

```diff
@@ -170,48 +170,46 @@ the cluster instead.
 Add user into the current app. Change current app by
 `skycli config set-app`.
 
-### skycli app view-tenant-config
+### skycli app view-user-config
 
-`skycli app view-tenant-config`
+`skycli app view-user-config`
 
-View the current app tenant config in yaml format.
+View the current app user config in yaml format.
 
 #### Example
 
 ```
-$ skycli app view-tenant-config
-- MASTER_KEY: xxxxx
-- API_KEY: xxxxx
-- SSO:
-  ALLOWED_CALLBACK_URLS:
+$ skycli app view-user-config
+masterkey: xxxxx
+apikey: xxxxx
+sso:
+  allowedcallbackurls:
   - http://127.0.0.1:5000/
   - http://localhost:5000/
-  MERGE_USERS_WITH_IDENTICAL_EMAIL: true
+  mergeuserswithidenticalemail: true
```

Comment:
We would allow camelCase right?

Reply:
No. If we allow camelCase then the developer can write `foobar` and `fooBar` in YAML legally but they conflict when we lowercase them.

Reply:
I don't think it's a good UX; it is hard for human to understand and does not look good. Can we validate key uniqueness after normalization instead?

Reply:
After a more throughout consideration, I am going to drop support for configuring via environment variable due to the following reason. It is hard to encode dictionary and list in environment variable, especially when the element is itself a struct. Instead, I am going to support configuring standalone mode via a YAML file.

Reply:
So rest of the spec will change back to _ separator?

Reply:
@louischan-oursky The implemented config has _ separators. Can you update the config key name to the correct format?

Reply:
Updated.